stdlib/LinearAlgebra/src/tridiag.jl

```diff
@@ -170,6 +170,9 @@ adjoint(S::SymTridiagonal) = Adjoint(S)
 Base.copy(S::Adjoint{<:Any,<:SymTridiagonal}) = SymTridiagonal(map(x -> copy.(adjoint.(x)), (S.parent.dv, S.parent.ev))...)
 Base.copy(S::Transpose{<:Any,<:SymTridiagonal}) = SymTridiagonal(map(x -> copy.(transpose.(x)), (S.parent.dv, S.parent.ev))...)
 
+ishermitian(S::SymTridiagonal) = isreal(S.dv) && isreal(S.ev)
```

Comment:
`SymTridiagonal` allows `ev` to have one extra element (i.e., `dv` and `ev` are allowed to have the same length). So, to be correct, should `isreal(S.ev)` be replaced by `isreal(view(S.ev, 1:length(S.dv)-1))`?